README.md

```diff
@@ -120,6 +120,25 @@ spring.datasource.username=DBUSERNAME
 spring.datasource.password=DBPASSWORD
 ```
 
+#### Garbage Collection
+In Java 11 added news Garbage Collection options to measuring, managing and better performance. 
+1. Epsilon : Epsilon removes the impact GC has on performance. You can use it to see how garbage collection affects your app's performance and what your memory threshold is since it'll tell you when it runs out.
```

Comment:
```suggestion
1. Epsilon : Epsilon removes the impact that the GC has on performance. You can use it to see how garbage collection affects your app's performance and what your memory threshold is, since it'll tell you when it runs out.
```